src/types/keyof.test.ts

```diff
@@ -60,7 +59,7 @@ describe(keyof, () => {
         ${'huh?'}  | ${Error('error in [YourAnswer]: expected one of the literals "yes", "maybe" or "no", got: "huh?"')}
     `('#translate($input) -> $output', ({ input, output }) => {
         if (output instanceof Error) {
-            expect(() => YourAnswer.translate(input)).toThrowWithMessage(ValidationError, output.message);
+            expect(() => YourAnswer.translate(input)).toThrow(output.message);
```

Comment:
`ValidationError` has a private constructor and that is no longer supported by `toThrowWithMessage`. The test was not that important imo, so just removed the instanceof check.

Reply:
We should still check if the error is an instance of `ValidationError`, because existing users may depend on this. 

fixed in commit
https://github.com/skunkteam/types/pull/73/files/1ab613b81edd1c83e3d427a3200252fd5aaee6d7